### PR TITLE
Fix USACO 992

### PR DESCRIPTION
**Pull request by Aadit-Ambadkar**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

#3008

### CODE REVIEW

**Review by ryanchou-dev — APPROVED** [review]
why does this warrant author credit?

**Review by bqi343 — CHANGES_REQUESTED** [review]
^

File: solutions/usaco-992.mdx [hunk 1]
```diff
@@ -72,17 +76,20 @@ int main () {
...
-			sol = max (sol, mid);
+			sol = max(sol, mid);
 			lo = mid + 1;
 		} else {
 			hi = mid - 1;
 		}
 	}
 
-	cout << (sol == top ? -1 : sol) << '\n';
+	cout << ((sol == top) ? -1 : sol) << '\n';
```
Comment (bqi343) [inline solutions/usaco-992.mdx:86]: why?

**Comment by Aadit-Ambadkar** [comment]
No it doesn't. I had other plans but didn't actually do them so I removed the author credit thingy.

File: solutions/usaco-992.mdx (hunk 1 above)
Comment (Aadit-Ambadkar) [reply solutions/usaco-992.mdx:86]: clarity

Comment (bqi343) [reply solutions/usaco-992.mdx:86]: this line doesn't even look like it's necessary? can just print `sol`

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Comment by bqi343** [comment]
oops my previous comment was wrong. I simplified the code not to use `sol` and resubmitted it to check.

**Review by bqi343 — APPROVED** [review]

**Pull request merged**